testing/py-virtualenvwrapper/APKBUILD

```diff
@@ -0,0 +1,50 @@
+# Contributor: Herve Beraud <herveberaud.pro@gmail.com>
+# Maintainer: Herve Beraud <herveberaud.pro@gmail.com>
+pkgname=py-virtualenvwrapper
+pkgver=4.8.0
+pkgrel=0
+pkgdesc="Enhancements to virtualenv"
+url="https://bitbucket.org/virtualenvwrapper/virtualenvwrapper/"
+arch="noarch"
+license="MIT"
+depends="python"
+makedepends='python2-dev python3-dev py-setuptools py-pbr'
+install=""
+subpackages=""
+source="${pkgname}-${pkgver}.tar.gz::https://bitbucket.org/virtualenvwrapper/virtualenvwrapper/get/${pkgver}.tar.gz"
+builddir="${srcdir}/virtualenvwrapper-virtualenvwrapper-c9300524289e"
+
+build() {
+	cd "${builddir}"
+	export PBR_VERSION=${pkgver}
+	python2 setup.py build || return 1
+	python3 setup.py build || return 1
+}
+
+package() {
+	mkdir -p "${pkgdir}"
+}
+
+_py2() {
+	replaces="${pkgname}"
+	depends="${depends//py-/py2-}"
+	_py python2
+}
+
+_py3() {
+	depends="${depends//py-/py3-}"
+	_py python3
+}
+
+_py() {
+	local python="$1"
+	pkgdesc="${pkgdesc} (for ${python})"
+	depends="${depends} ${python}"
+	install_if="${pkgname}=${pkgver}-r${pkgrel} ${python}"
+
+	cd "${builddir}"
+	export PBR_VERSION=${pkgver}
+	python setup.py install --prefix=/usr --root="${pkgdir}" --skip-build
```

Comment:
`python` is symlink to python2. You need to use `$python` here (contains python2 or python3).

There should be `$subpkgdir`, not `$pkgdir`.